packages/manager/modules/log-to-customer/src/live-tail/component.js

```diff
@@ -5,11 +5,11 @@ export default {
   controller,
   template,
   transclude: {
-    tile: '?rightTile',
+    serviceDescription: '?serviceDescription',
```

Comment:
Can be simplified
```suggestion
    serviceDescription: '?',
```

Reply:
Already suggested previously but it actually doesn't work